### ISSUE TITLE
manual/scalaGuide/main/sql/ScalaDatabase.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#621]
- https://github.com/garbagetown/playdocja/blob/2.2.0/documentation/2.2.0/manual/scalaGuide/main/sql/ScalaDatabase.md

```
--- /Users/garbagetown/Desktop/2.1.5/manual/scalaGuide/main/sql/ScalaDatabase.md    2013-09-19 22:53:02.000000000 +0900
+++ /Users/garbagetown/Desktop/2.2.0/manual/scalaGuide/main/sql/ScalaDatabase.md    2013-09-19 10:11:22.000000000 +0900
@@ -2,9 +2,18 @@

 ## Configuring JDBC connection pools

-Play 2.0 provides a plug-in for managing JDBC connection pools. You can configure as many databases as you need.
+Play provides a plug-in for managing JDBC connection pools. You can configure as many databases as you need.

-To enable the database plug-in, configure a connection pool in the `conf/application.conf` file. By convention, the default JDBC data source must be called `default` and the corresponding configuration properties are `db.default.driver` and `db.default.url`.
+
+To enable the database plug-in, add jdbc in your build dependencies :
+
+'''scala
+val appDependencies = Seq(
+  jdbc
+)
+'''
+
+Then you must configure a connection pool in the `conf/application.conf` file. By convention, the default JDBC data source must be called `default` and the corresponding configuration properties are `db.default.driver` and `db.default.url`.

 If something isn’t properly configured you will be notified directly in your browser:

@@ -26,7 +35,7 @@
 db.default.url="jdbc:h2:/path/to/db-file"
 '''

-The details of the H2 database URLs are found from [[H2 Database Engine Cheat Sheet |http://www.h2database.com/html/cheatSheet.html]].
+The details of the H2 database URLs are found from [H2 Database Engine Cheat Sheet](http://www.h2database.com/html/cheatSheet.html).

 ### SQLite database engine connection properties

@@ -76,7 +85,7 @@

 ## Configuring the JDBC Driver

-Play 2.0 is bundled only with an [[H2 | http://www.h2database.com]] database driver. Consequently, to deploy in production you will need to add your database driver as a dependency.
+Play is bundled only with an [H2](http://www.h2database.com) database driver. Consequently, to deploy in production you will need to add your database driver as a dependency.

 For example, if you use MySQL5, you need to add a [[dependency | SBTDependencies]] for the connector:

@@ -136,4 +145,4 @@
 }
 '''

-> **Next:** [[Using Anorm to access your database | ScalaAnorm]]
\ No newline at end of file
+> **Next:** [[Using Anorm to access your database | ScalaAnorm]]

```

**Comment by garbagetown** [comment]
pull reqed at #698

**Comment by wm3** [comment]
merged #698
